tools/lkl/lib/virtio_net_dpdk.c

```diff
@@ -402,8 +403,9 @@ struct lkl_netdev *lkl_netdev_dpdk_create(const char *ifparams, int offload,
 	if (offload & (BIT(LKL_VIRTIO_NET_F_GUEST_TSO4) |
 			BIT(LKL_VIRTIO_NET_F_GUEST_TSO6) |
 			BIT(LKL_VIRTIO_NET_F_MRG_RXBUF))) {
-		portconf.rxmode.enable_lro = 1;
-		portconf.rxmode.hw_strip_crc = 1;
+		// TODO check rx_offload_capa before applying this
+		portconf.rxmode.offloads |= DEV_RX_OFFLOAD_TCP_LRO;
+		portconf.rxmode.offloads |= ~DEV_RX_OFFLOAD_KEEP_CRC;
```

Comment:
I am only unsure about this line. There is now `hw_strip_crc` option any more but just negating this might be also not the solution.